### DISCUSSION TITLE
Link thumbnail to the target page

### DISCUSSION THREAD

**Discussion by jochenfritz** [microsoft-search/pnp-modern-search#2486] [Q&A]
I have a question regarding the current 'cards' layout of the search web part. 
The thumbnail is currently not linked to the target page. However, an iFrame can be set to preview the target page, but my users expect to access the page also via the image and not just via the title.
Can the link to the target page be set via the managed card fields?

**Comment by wobba** [comment]
You can customize the template and add an <a> tag around the document card.

**Comment by jochenfritz** [comment]
@wobba Thank you for your answer. Can you please give me a more concrete hint how to set this up?

**Reply by wobba** [reply]
Pick the template you want to start with, then click the `{}` icon which will copy this as the base template, and edit the html to match your needs.

![image](https://user-images.githubusercontent.com/51104/199232340-6ac1014c-14de-47ff-8dc9-ef8da80139c8.png)
